### DISCUSSION TITLE
突然登入不了

### DISCUSSION THREAD

**Discussion by WhiteCloud0106** [Lost-MSth/Arcaea-server#146] [Q&A] [answered]
今天早上要登入的時候用到這個狀況
![Screenshot_20240129_094346](https://github.com/Lost-MSth/Arcaea-server/assets/94116682/cefac2a4-3d6c-414e-9ad4-131983ba7b5e)
能登出的都登出了，也都重啟過但還是登不進去😰

**Comment by Lost-MSth** [comment, marked as answer]
有服务端日志吗？没有就是连到了官服，客户端重启，重输账号密码

**Reply by WhiteCloud0106** [reply]
我按登入的時候服務端訊息和一般登入的訊息是一樣的
之後我先登了另一個帳戶再改登入這個帳戶就可以了
感覺蠻詭異的

**Reply by BakaGzRay12456** [reply]
重新输入账号密码就行
